relay-event-schema/src/protocol/user.rs

```diff
@@ -76,6 +76,10 @@ pub struct User {
     #[metastructure(skip_serialization = "empty")]
     pub segment: Annotated<String>,
 
+    /// The user string representation as handled in Sentry.
```

Comment:
nit: Could you add some examples here (as in the `get_event_user_tag` doc comment)?

Reply:
Yess, will do. Should I also rename the function to `get_event_user_tag` it bugs me as it is written rn.